opengapps-packages.mk

```diff
@@ -1,3 +1,4 @@
+include vendor/google/build/config.mk
```

Comment:
I have literally 0% experience with these .mk files, but the line you reference includes a minus-sign before the include, could that have any effect?